core/arch/arm/plat-stm/tz_a9init.S

```diff
@@ -368,10 +368,10 @@ loop_1:
 	movt r1, #(CPU_PORT_FILT_START >> 16)
 	str  r1, [r0, #PL310_ADDR_FILT_START]
 
-	/* Allow NSec to manage FIQ/Imprecise abort */
-	mrc p15, 0, r0, c1, c1, 0    /* read Secure Configuration Register */
-	orr r0, r0, #0x30            /* SCR[FW]=1, SCR[AW]=1 */
-	mcr p15, 0, r0, c1, c1, 0    /* write updated value in Secure Configuration Register */
+	/* Allow NSec to manage FIQ/Imprecise abort (SCR[FW]=1, SCR[AW]=1) */
+	mrc p15, 0, r0, c1, c0, 0
+	orr r0, r0, #0x30            /*  */
+	mcr p15, 0, r0, c1, c0, 0    /* write updated value in Secure Configuration Register */
```

Comment:
i think. but having pushed such a change (read/write SCTRL instead of SCR) makes me doubt on my capabilities :(

actually i will push for 2 commit : bring cannes and more useful (to us) the b2260 (96board with a cannes2).